packages/storage-driver-local/src/index.ts

```diff
@@ -75,10 +75,14 @@ export class DriverLocal implements Driver {
 	}
 
 	async write(filepath: string, content: Readable) {
-		const fullPath = this.fullPath(filepath);
-		await this.ensureDir(dirname(fullPath));
-		const writeStream = createWriteStream(fullPath);
-		await pipeline(content, writeStream);
+		try {
+			const fullPath = this.fullPath(filepath);
+			await this.ensureDir(dirname(fullPath));
+			const writeStream = createWriteStream(fullPath);
+			await pipeline(content, writeStream);
+		} catch (error) {
+			throw new Error(`Failed writing file "${filepath}": ${error}`);
+		}
```

Comment:
Actually, I think we should revert all changes here, seeing that none of the other storage drivers have a `try...catch` statement in the `write` method. I assume the idea is to catch the error at the places where the method is called instead.

Otherwise we should apply this to all other drivers as well.